awscli/customizations/wizard/app.py

```diff
@@ -70,6 +75,9 @@ def get_current_prompt_choices(self):
             return [choice['display'] for choice in choices]
         return None
 
+    def is_current_prompt_has_details(self):
```

Comment:
Let's just remove the `is` on this method name. It is a little hard to read when there is both an `is` and `has` in the method.